src/JSInterop/Microsoft.JSInterop/src/JSRuntime.cs

```diff
@@ -174,6 +183,46 @@ protected virtual void BeginInvokeJS(long taskId, string identifier, string? arg
             DotNetInvocationInfo invocationInfo,
             in DotNetInvocationResult invocationResult);
 
+        /// <summary>
+        /// Invoked by JS
```

Comment:
Is this comment right? I thought this was invoked by .NET code in the JSON serializer. If so, suggestion:

```suggestion
        /// Invoked by the platform to transfer a byte array from .NET to JS.
```